.github/CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @EnoRage @catenocrypt @hewigovens @kolya182 @prazd
```

Comment:
why delete this ?

Reply:
Hmm, file got deleted unintentionally when I merged master branch, strange.